<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xqInfo_User</class>
 <widget class="QMainWindow" name="xqInfo_User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">#frame{border:2px solid rgb(170,85,0)}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>130</width>
                 <height>41</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>130</width>
                 <height>41</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>小区名称  ：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>130</width>
                 <height>41</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>130</width>
                 <height>41</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>负责人姓名：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="minimumSize">
                <size>
                 <width>130</width>
                 <height>41</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>130</width>
                 <height>41</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>负责人职位：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>130</width>
                 <height>41</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>130</width>
                 <height>41</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>负责人性别：：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="minimumSize">
                <size>
                 <width>130</width>
                 <height>41</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>130</width>
                 <height>41</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>联系方式  ：</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineEdit_xqname">
               <property name="minimumSize">
                <size>
                 <width>441</width>
                 <height>55</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>441</width>
                 <height>51</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_fzrname">
               <property name="minimumSize">
                <size>
                 <width>441</width>
                 <height>55</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>441</width>
                 <height>51</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_fzrzw">
               <property name="minimumSize">
                <size>
                 <width>441</width>
                 <height>55</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>441</width>
                 <height>51</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_sex">
               <property name="minimumSize">
                <size>
                 <width>441</width>
                 <height>55</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>441</width>
                 <height>51</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_tel">
               <property name="minimumSize">
                <size>
                 <width>441</width>
                 <height>55</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>441</width>
                 <height>51</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>41</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>  小区介绍</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
